src/flow.c

```diff
@@ -378,6 +378,89 @@ static inline void FlowUpdateEthernet(ThreadVars *tv, DecodeThreadVars *dtv,
     }
 }
 
+#if defined(ENABLE_ETM)
+#include <math.h>
```

Comment:
system includes should go into `suricata-common.h` and if they are new they should be checked in `configure.ac`.